src/platform/plugins/private/links/public/components/editor/links_editor.tsx

```diff
@@ -195,6 +222,16 @@ const LinksEditor = ({
               legend={LinksStrings.editor.panelEditor.getLayoutSettingsLegend()}
             />
           </EuiFormRow>
+          <EuiFormRow label={LinksStrings.editor.linkEditor.getLinkTextOverflowLabel()}>
+            <EuiButtonGroup
+              legend={LinksStrings.editor.linkEditor.getLinkTextOverflowLegend()}
+              type="single"
+              options={toggleTextOverflowOptions}
+              idSelected={currentTextOverflow}
+              buttonSize="compressed"
+              onChange={(id) => setCurrentTextOverflow(id as LinksTextOverflowType)}
+            />
+          </EuiFormRow>
```

Comment:
It feels like we are giving the "Text overflow" setting too much importance by putting it near the top 😅 It feels strange moving it below the "Links" section as well, though....

| This PR | Suggested |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/953daa18-7fda-452d-9114-32331ad131b9) | ![image](https://github.com/user-attachments/assets/e899509b-3a12-4048-9e28-18e32add0623) | 

Any suggestions here @andreadelrio?

Reply:
I was thinking the same thing. Another thing that feels off and I don't think there's much we can do about is that you can't see the preview of the text overflow setting you're choosing. I think I would lean towards moving this setting to the bottom.

Reply:
💯 We really need to update links to be inline editable with a push flyout - this is actually a Spacetime project I'm considering 👀